package.json

```diff
@@ -53,11 +53,14 @@
     "jest-serializer-ansi": "^1.0.3",
     "jest-watch-select-projects": "^2.0.0",
     "jsdom": "^16.4.0",
-    "kcd-scripts": "^7.5.1",
+    "kcd-scripts": "^7.5.3",
     "typescript": "^4.1.2"
   },
   "eslintConfig": {
-    "extends": "./node_modules/kcd-scripts/eslint.js",
+    "extends": [
+      "./node_modules/kcd-scripts/eslint.js",
+      "plugin:import/typescript"
+    ],
```

Comment:
As recommended: https://github.com/benmosher/eslint-plugin-import#typescript

Reply:
Yes I see, the strange things is that is should already be in kcd

Reply:
It should be. That's an oversight